hack/self-signed-ca.sh

```diff
@@ -116,16 +116,32 @@ caBundle=$(cat ${tmpdir}/ca.crt | openssl enc -a -A)
 echo "Encoded CA:"
 echo -e "${caBundle} \n"
 
-# Patch CA Certificate to webhooks
-mutatingPatchString='[{"op": "replace", "path": "/webhooks/0/clientConfig/caBundle", "value":"{{CA_BUNDLE}}"}, {"op": "replace", "path": "/webhooks/1/clientConfig/caBundle", "value":"{{CA_BUNDLE}}"}]'
+# Patch CA Certificate to mutatingWebhook
+mutatingWebhookCount=$(kubectl get mutatingwebhookconfiguration | grep ${webhookConfigName} | awk '{print $2}')
```

Comment:
how about using the following command to get the count?
`kubectl get mutatingwebhookconfiguration inferenceservice.serving.kubeflow.org -ojson | jq -r '.webhooks' | jq length`

Reply:
this is better, but if jq is not installed in the system, the scripts should exits or fallback using awk？

Reply:
we can exit the script so user know it needs to install jq.